tools/fiptool/fiptool.c

```diff
@@ -561,10 +570,29 @@ static void parse_plat_toc_flags(char *arg, unsigned long long *toc_flags)
 	*toc_flags |= flags << 32;
 }
 
+static int is_power_of_2(unsigned long x)
+{
+        return x && !(x & (x - 1));
+}
```

Comment:
I am inclined to go with masahir0y here.  I don't think we should try and generalize this proposition.  Even though utils.h is and will probably remain target agnostic, I don't think we should encourage sharing of code between TF and the host tools (as masahir0y already mentioned, we already share some code but this is mostly for convenience as the alternatives require more work).